src/acquisition/rvdss/rvdss_historic.py

```diff
@@ -0,0 +1,652 @@
+
+    if matches:
+        season = matches.group(0)
+    years=season.split("-")
+    return(years)
+
+def append_urls(urls):
+    # Add https to the urls
+    for i in range(len(urls)):
+        temp_url = urls[i]
+  
+        http_present = re.search("http",temp_url)
+        if not http_present:
+            urls[i]="https://www.canada.ca"+temp_url
+    return(urls)
+
+def report_urls(soup):
+    # Get links for individual weeks
+    year= "-".join(get_report_season(soup))
+    links=soup.find_all('a')
+    alternative_url = "http://www.phac-aspc.gc.ca/bid-bmi/dsd-dsm/rvdi-divr/"+year
+    
+    urls = [link.get("href") for link in links if "ending" in str(link) or 
+            alternative_url in str(link)]
+    
+    report_links = append_urls(urls)
+    return(report_links)
+
+def report_weeks(soup):
+    links=soup.find_all('a')
+    full_weeks = [link.text for link in links if "Week" in str(link)]
+    weeks= [int(re.search('Week (.+?) ', week).group(1)) for week in full_weeks]
+    return(weeks)
+
+def get_report_date(week,start_year,epi=False):
+    if week < 35: 
+        year=int(start_year)+1
+    else:
+        year=int(start_year)
+    
+    epi_week =  Week(year, week)
+    
+    if epi==False:
+        report_date = str(epi_week.enddate())
+    else:
+        report_date =  str(epi_week)
+        
+    return(report_date)
+
+def abbreviate_virus(full_name):
+    lowercase=full_name.lower()
+    
+    if any(name in lowercase for name in ["parainfluenza","para","piv"]):
+        if "hpiv" not in lowercase:
+            abbrev = re.sub("parainfluenza|para|piv","hpiv",lowercase)
+        else:
+            abbrev = lowercase
+    elif any(name in lowercase for name in ["adenovirus","adeno"]):
+        abbrev =  re.sub("adenovirus|adeno","adv",lowercase)
+    elif "human metapneumovirus" in lowercase:
+        abbrev =  re.sub("human metapneumovirus","hmpv",lowercase)
+    elif any(name in lowercase for name in ["enterovirus/rhinovirus","rhinovirus","rhv","entero/rhino","rhino","ev/rv","evrv"]):
+        abbrev = re.sub("enterovirus/rhinovirus|rhinovirus|rhv|entero/rhino|rhino|ev/rv|evrv","ev_rv",lowercase)
+    elif any(name in lowercase for name in ["coronavirus","coron","coro"]):
+        abbrev = re.sub("coronavirus|coron|coro","hcov",lowercase)
+    elif "respiratory syncytial virus" in lowercase:
+        abbrev = re.sub("respiratory syncytial virus","rsv",lowercase)
+    elif "influenza" in lowercase:
+        abbrev = re.sub("influenza","flu",lowercase)       
+    elif "sarscov2" in lowercase:
+        abbrev = re.sub("sarscov2","sars-cov-2",lowercase) 
+    else:
+        abbrev=lowercase
+    return(abbrev)
+
+def abbreviate_geo(full_name):
+    lowercase=full_name.lower()
+    
+    if "newfoundland" in lowercase:
+        abbrev =  "nl"
+    elif "prince edward island" in lowercase:
+        abbrev =  "pe"
+    elif "nova scotia" in lowercase:
+        abbrev =  "ns"
```

Comment:
suggestion: We'll want this to be in a dictionary in `constants.py`, like [this](https://github.com/cmu-delphi/delphi-epidata/blob/dev/src/acquisition/fluview/fluview_locations.py#L17).